src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -199,8 +199,8 @@
     <script type="text/template" id="template-legend-item-multiple">
         <div class="legend-layer multiple">
             <div class="legend-title"><%- layerName %></div>
-            <% _.each(legend, function(item) { %>
-                <div class="item">
+            <% _.each(legend, function(item, index) { %>
+                <div class="item <% if (index > 4) { %>extra<% } %>">
```

Comment:
What do you think @jfrankl? I think this makes sense, but wanted to check with you first.